templates/kafka.service.erb

```diff
@@ -0,0 +1,25 @@
+[Unit]
+Description=publish-subscribe messaging rethought as a distributed commit log
+
+[Service]
+Type=simple
+TimeoutStopSec=<%= scope.lookupvar('kafka::server::kill_timeout_seconds') %>
+
+Environment=ENABLED=no
```

Comment:
when will this change to 'yes'? otherwise won't the pre-start script always not start kafka?

Reply:
The ENABLED value should be overridden in `EnvironmentFile=/etc/default/kafka` I think.

I tried to copy as much as possible how the upstart script works:
https://github.com/Yelp/puppet-kafka/blob/master/templates/kafka.conf.erb#L10

Reply:
Oh I see. I think it should be okay then, but I'm not an expert on that so make sure to test haha